src/GraphQL.Tests/Bugs/Bug1773.cs

```diff
@@ -51,40 +51,45 @@ public void nonnull_list_valid()
         [Fact]
         public void nonnull_list_throws_when_null()
         {
-            AssertQueryWithError("{testListNullInvalid}", "{\"testListNullInvalid\": null}", "Error trying to resolve field 'testListNullInvalid'.", 1, 2, new[] { "testListNullInvalid" },
-                new InvalidOperationException("Cannot return a null member within a non-null list for list index 0."));
+            AssertQueryWithError("{testListNullInvalid}", "{\"testListNullInvalid\": null}", "Error trying to resolve field 'testListNullInvalid'.", 1, 2, new object[] { "testListNullInvalid", 0 },
+                new InvalidOperationException("Cannot return null for a non-null type. Field: testListNullInvalid, Type: Int!."));
```

Comment:
This message is confusing a bit, `testListNullInvalid` has `[Int!]` type, not `Int!`. See https://github.com/graphql-dotnet/graphql-dotnet/pull/2341/files#r587278824

Reply:
See #2354